### DISCUSSION TITLE
Lesson 3: AI Foundry

### DISCUSSION THREAD

**Discussion by Booker-CU** [Cyfrin/foundry-full-course-cu#1614] [General]
I just want to say thanks for this section. It means a lot.
Especially the 15/20 of trying to figure something out. 

Before AI came out last year I would spend hours/days when I got stuck cause I wasn't sure where to go to get help. I like that you say to give yourself a time limit. I would spiral for hours and be super frustrated. I think that slowed down my overall progress because it felts so frustrating. I'm glad I can get some help/tips from AI when needed + other places!!!

**Comment by EngrPips** [comment]
Hello @Booker-CU, It is interesting to hear how easily you have been unblocking yourself. Keep crushing it, and yeah, Patrick did give us a great head-on using AI buddy whenever we get blocked, and that helped

**Reply by Booker-CU** [reply]
yes he sure did :) ... I also really appreciate the idea of giving yourself a time limit to not drive yourself crazy... 15 to 20mins...
its very easy to spiral out of control when your learning and have no idea how to fix a bug or error that you are getting

**Reply by EngrPips** [reply]
Programming can sometimes be challenging.
